startupscript/bash-completion.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
It would be great if each of these smaller files had a header comment which provided a little bit about the context in which they are called. For example:

```
# bash-completion.sh
#
# Writes a block to the user's ~/.bashrc file such that Bash shells pick up
# any Bash completion scripts that have been installed.
#
# Note that this script is intended to be source from the "post-startup.sh" script
# and is dependent on some functions and variables already being set up:
#
# - emit (function)
# - USER_BASHRC: path to user's ~/.bashrc file
```